### PR TITLE
Fax numbers for gsync

### PR DESCRIPTION
**Pull request by sampatbadhe**
Hello Team,

This PR contain following changes -
- returns fax number in response.
- returns google type same as displayed in google.
- fix failing specs

Please review and advice.

Thanks.

### CODE REVIEW

File: lib/google-contacts/element.rb [hunk 1]
```diff
@@ -289,7 +309,10 @@ def update_groups(*group_links)
     end
 
     private
-     # Evil ahead
+      def get_google_type(google_type)
```
Comment (rohandaxini) [inline lib/google-contacts/element.rb]: @sampatbadhe should this method name be more meaningful? What is google_type ?

Comment (sampatbadhe) [inline lib/google-contacts/element.rb]: hello @rohandaxini 
This returns the label name which google used for `phone number`, `address`, `email address`...

Comment (rohandaxini) [inline lib/google-contacts/element.rb]: @sampatbadhe should it be `get_google_label_name` or anything else that is more relevant? Method name `get_google_type` is very confusing and its not clear what it returns or does. It just has comment `# Evil ahead`

**Comment by rohandaxini** [comment]
@sampatbadhe rest all is fine :+1:

File: lib/google-contacts/element.rb (hunk 1 above)
Comment (sampatbadhe) [inline lib/google-contacts/element.rb]: @rohandaxini sure I'll change method name to `get_google_label_name`.

`# Evil ahead` this comment was there previously. I have removed it in my changes. :stuck_out_tongue_winking_eye:

**Comment by sampatbadhe** [comment]
Thanks @rohandaxini

**Comment by rohanpujaris** [comment]
:+1:

**Comment by titame** [comment]
:+1:

**Comment by abhijitsinha** [comment]
Hi @rohandaxini - this PR is good to merge to master. But please **do not delete** the branch as this branch has been referenced in Gemfile and used in `production` branch.

**Comment by rohandaxini** [comment]
Sure @abhijitsinha

**Pull request merged**